SCIM doc updates (#7309)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/manage-access/scim.md

    <Lightbox src="/img/docs/dbt-cloud/access-control/enable-scim.png" width="60%" title="SCIM enabled in the Okta configuration settings." />
4. Record the **SCIM base URL** field for use in a later step.
5. Click **Create SCIM token**.
    :::note
    

>     It's best practice to rotate your SCIM token at regular intervals. To do so, follow these same instructions to create a new one. Be sure not to delete the old token until it has been replaced in your IdP.

Author: ```suggestion
    It's best practice to rotate your SCIM token at regular intervals. To do so, follow these same instructions to create a new one. To avoid service disruptions, don't delete the old token in dbt Cloud until you've replaced it in your IdP.
```

>     It's a best practice to rotate your SCIM token at regular intervals. You can follow these same instructions to rotate the token by creating a new one. Be sure not to delete the old token until it has been replaced in your IdP.

Author: ```suggestion
    It's best practice to rotate your SCIM token at regular intervals. To do so, follow these same instructions to create a new one. Be sure not to delete the old token until it has been replaced in your IdP.
```

>     It's best practice to rotate your SCIM token at regular intervals. To do so, follow these same instructions to create a new one. To avoid service disruptions, don't delete the old token in dbt Cloud until you've replaced it in your IdP.

Reviewer: ```suggestion
    To follow best practices, you should regularly rotate your SCIM tokens. To do so, follow these same instructions you did to create a new one. To avoid service disruptions, remember to replace your token in your IdP before deleting the old token in <Constant name="cloud" />.
```
